src/commands/eval.ts

```diff
@@ -194,9 +194,32 @@ export async function doEval(
     const shareableUrl =
       cmdObj.share && config.sharing ? await createShareableUrl(evalRecord) : null;
 
-    const summary = await evalRecord.toEvaluateSummary();
-    const table = await evalRecord.getTable();
-    if (cmdObj.table && getLogLevel() !== 'debug') {
+    let successes = 0;
+    let failures = 0;
+    const tokenUsage = {
+      total: 0,
+      prompt: 0,
+      completion: 0,
+      cached: 0,
+    };
+
+    for (const prompt of evalRecord.prompts) {
+      if (prompt.metrics?.testPassCount) {
+        successes += prompt.metrics.testPassCount;
+      }
+      if (prompt.metrics?.testFailCount) {
+        failures += prompt.metrics.testFailCount;
+      }
+      tokenUsage.total += prompt.metrics?.tokenUsage?.total || 0;
+      tokenUsage.prompt += prompt.metrics?.tokenUsage?.prompt || 0;
+      tokenUsage.completion += prompt.metrics?.tokenUsage?.completion || 0;
+      tokenUsage.cached += prompt.metrics?.tokenUsage?.cached || 0;
+    }
+    const totalTests = successes + failures;
+    const passRate = (successes / totalTests) * 100;
```

Comment:
We can't just get the table which has all the results so we'll sum them up off of the prompts